### DISCUSSION TITLE
Gamma correction when resizing images

### DISCUSSION THREAD

**Discussion by lejsekt** [SixLabors/ImageSharp#1736] [Q&A] [answered]
Hi,
is it possible to adjust gamma correction when resizing images?
As described here: http://www.ericbrasseur.org/gamma.html?i=1
I've tried resizing the last image:
![gamma-1 0-or-2 2](https://user-images.githubusercontent.com/15654203/129159584-e0aed15a-c6c7-439f-bd4a-20a48746c71b.png)

But didn't get the expected results :)
![nearest-neighbour](https://user-images.githubusercontent.com/15654203/129159540-82ff0b2b-8e2c-4672-8802-404018c8e697.jpg)

**Comment by JimBobSquarePants** [comment, marked as answer]
Yeah there is 😄 

https://docs.sixlabors.com/api/ImageSharp/SixLabors.ImageSharp.Processing.ResizeOptions.html#SixLabors_ImageSharp_Processing_ResizeOptions_Compand

**Reply by lejsekt** [reply]
Thanks for a quick reply!
Which of the options in resize options should be set? Still can't get it to work.

**Reply by JimBobSquarePants** [reply]
Compand. You want that to be true

**Reply by JimBobSquarePants** [reply]
```c#
using (var source = Image.Load<Rgba32>(Path.Combine(inPath, "gamma.png")))
{
    var ro = new ResizeOptions { Compand = true, Size = new Size(source.Size().Width / 2, 0) };
    source.Mutate(x => x.Resize(ro));
    source.Save(Path.Combine(outPath, "gamma-resize.png"));
}
```

![gamma-resize](https://user-images.githubusercontent.com/385879/129188201-a7f4b51e-156d-4a6a-afdd-99acfc04510b.png)

**Reply by lejsekt** [reply]
Thanks!
